ivy/functional/frontends/numpy/statistics/order_statistics.py

```diff
@@ -35,6 +39,15 @@ def _cpercentile(N, percent, key=lambda x: x):
     return d0 + d1
 
 
+@to_ivy_arrays_and_back
+@handle_numpy_out
+def ptp(a, axis=None, out=None, dtype=None, keepdims=False):

```

Comment:
This function signature is contrary to what indicated in the numpy documentation
```python
def numpy.ptp(a, axis=None, out=None, keepdims=<no value>):
    pass
```suggestion
def ptp(a, axis=None, out=None, keepdims=False):

```
```